web/src/views/DandiMetaViewer.vue

```diff
@@ -76,26 +102,22 @@
                     <v-list-item-content>{{item}}</v-list-item-content>
                   </v-list-item>
                 </template>
+                <template v-for="(item, k) in extraFields">
+                  <v-subheader :key="k">{{ k }}</v-subheader>
+                  <v-list-item :key="k">
+                    <v-list-item-content>
+                      <template v-if="['object', 'array'].includes(schema.properties[k].type)">
+                        <vue-json-pretty :data="item" highlightMouseoverNode />
```

Comment:
Aren't props usually passed as e.g. `highlight-mouseover-node`? Or is my Vue-fu just stale?

Reply:
Nope your Vue-fu is still good